package.json

```diff
@@ -57,7 +57,7 @@
     "passport-twitter": "^1.0.2",
     "phantomjs": ">=1.9.0",
     "serve-favicon": "^2.3.0",
-    "socket.io": "^1.3.5",
+    "socket.io": "Automattic/socket.io",
```

Comment:
@codydaig pointed out that the `socket.io-client` package isn't installed with `socket.io` if the stand-alone `socket.io-client` is installed. So given that behavior, my above proposal may not be viable. 

Perhaps, we will just have to wait for engine.io-client to update the npm package to their latest version. I wouldn't expect to be waiting too long for this to happen. 

In the meantime, we could work on building these tests & polishing up this PR in anticipation for the engine.io-client project to be updated.